### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by ZechCodes** [beginner-codes/project-ideas#12] [General]
## 👋 Welcome!

This GitHub repository is intended to be a collection of project ideas anyone can use for inspiration. The projects are intended to help beginners grow their skills and let everyone else hone their abilities.

## 🎙 Discussion

Feel free to discuss the projects, ask any questions, and even show off! If you'd like to contribute feel free to submit a pull request!

All discussion will be moderated in accordance with our [Code of Conduct](https://github.com/beginnerpy-com/project-ideas/blob/main/CODE_OF_CONDUCT.md)

## 📝 Project Ideas

You can see the full list of project ideas in the [README.md](https://github.com/beginnerpy-com/project-ideas/blob/main/README.md)

## 💬 Join Us On Discord

We're a community of coders who believe the best way to grow is to help others learn. **[Join us on Discord!!!](https://discord.gg/sfHykntuGy)**

**Comment by MusaKhan16** [comment]
Hello!

**Reply by ZechCodes** [reply]
Hi! How’s it going?

**Comment by MusaKhan16** [comment]
All good!
